### DISCUSSION TITLE
kmer size selection

### DISCUSSION THREAD

**Discussion by INFINITY1993** [ablab/spades#948] [Q&A] [answered]
Hi,
How can we know the kmer sizes we select are suitable for the sample? Are there any indicators that could give us clue?
The spades.log is attached.
Thanks
[spades.log](https://github.com/ablab/spades/files/8236897/spades.log)
!

**Comment by asl** [comment, marked as answer]
There is no definite rule or single criterion for this. Even more, different k-mer lengths might be deemed "optimal" depending on the criteria used. So, you'd simply analyze your data and decide if they are good enough. If you're unsure – stick to the defaults.

**Reply by INFINITY1993** [reply]
ok，thanks
